README.md

```diff
@@ -27,10 +27,14 @@ To build locally and install, refer [here](#build).
 View default configurations:
 ```bash
 $ sudo snap get openthread-border-router 
-Key        Value
-infra-if   wlan0
-radio-url  spinel+hdlc+uart:///dev/ttyACM0
-thread-if  wpan0
+Key             Value
+infra-if        wlan0
+radio-url       spinel+hdlc+uart:///dev/ttyACM0
+thread-if       wpan0
+
+interface-name  wpan0
+listen-address  "0.0.0.0"
+port            80
```

Comment:
The snap has multiple apps, and the given naming doesn't indicate that these belong to the web component only.

Reply:
options: 
- `web-listen-address` and `web-port`
- `gui-listen-address` and `gui-port`

Reply:
IMO, `webgui-bind-address` and `webgui-bind-port`.

Reply:
The `webgui-listen-address` clearly communicates the meaning of the variable and aligns with the upstream [SDK](https://github.com/openthread/ot-br-posix/blob/main/src/web/main.cpp#L141). However, if you prefer `webgui-bind-address` I can make the change accordingly.

Reply:
Bind Address and Bind Port are technically correct and consistent with each other. It is okay to follow the server's naming.